Allow PHP8

T2L opened the pull request.

T2L commented:
Alright, the last thing that is missing https://github.com/phpmd/phpmd, which is not PHP8 compatible because one of it's dependencies (https://github.com/pdepend/pdepend) is not compatible, but work is in progress: https://github.com/pdepend/pdepend/pull/498

michalbundyra reviewed (commented):
It looks good! Thanks for your effort.

I think we can ignore phpmd issues on PHP 8.0 for now.

Would you be able to rebase and update also phpunit workflow to use PHP 8.0? I've added github actions recently, and at some point we will drop Travis.

I've tried to push to your branch, but it looks like you've locked that. Thanks again

T2L commented on .travis.yml:
```diff
@@ -5,6 +5,7 @@ cache:
     - $HOME/.composer/cache
 
 php:
+  - nightly
```
PHP8.0 was not available as `8.0` at the time of this PR, but now it is https://travis-ci.community/t/php-8-0-missing/10132/27

T2L commented:
- The fork has been synced and rebased. I think you cannot push to it because the repo is part of the organization and not my personal Github
- Changed `nightly` to `8.0` on Travis
- Added `8.0` to Github Actions (BTW, even `8.1` can be added)

michalbundyra commented:
Thanks, @T2L!

T2L commented:
Any news regarding the release plan?

michalbundyra commented:
Yes, later today will be released :)
